lib/scrabble_class.rb

```diff
@@ -34,7 +37,6 @@ def self.score(word)
     end
 
 
-
     def self.highest_score_from(array_of_words)
       max_score = 0
       max_word = ""
```

Comment:
I'm a bit confused by the logic in your `if` statement below starting on line 45. It seems like every conditional within this conditional has the same block which reassigns the `max_score` and `max_word` variable.